framework/requirements.txt

```diff
@@ -22,3 +23,6 @@ pytest-timeout==2.2.0
 
 # Requirements for the report
 markdown==3.5.2
+
+# Requirements for the session
+cryptography==42.0.4

```

Comment:
TLS module is at 42.0.7.  Is there any reason to keep this at a lower version?

Reply:
This has now been upgraded.